internal/connect/command_custom_plugin_create.go

```diff
@@ -83,9 +89,11 @@ func (c *customPluginCommand) createCustomPlugin(cmd *cobra.Command, args []stri
 	if extension != "zip" && extension != "jar" {
 		return fmt.Errorf(`only file extensions ".jar" and ".zip" are allowed`)
 	}
+	cloudProvider := strings.ToLower(cloud)
 
 	request := *connectcustompluginv1.NewConnectV1PresignedUrlRequest()
 	request.SetContentFormat(extension)
+	request.SetCloud(cloudProvider)
```

Comment:
Should we be setting this value if it is `""`?

Reply:
Since we will check here: https://github.com/confluentinc/cc-connect-plugin-service/blob/master/api/v2/middleware.go#L42. It will be ok if the input is ""

Reply:
Great, can we combine the three lines above into one? I know this isn't part of your PR, but we don't recommend using `NewConnectV1PresignedUrlRequest()`. Instead, you should just initialize `request` as a single struct.